linux/flatpak/appstream-metainfo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+cat << EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mozilla.vpn</id>
+
+  <name>Mozilla VPN</name>
+  <summary>A fast, secure and easy to use VPN. Built by the makers of Firefox</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <content_rating type="oars-1.0" />
+
+  <developer id="org.mozilla">
+    <name>Mozilla Corporation</name>
+  </developer>
+
+$(python $(dirname $0)/appstream-releases.py | sed s'/^/  /g')
```

Comment:
nitpick: Maybe move this outside the file content into a variable so it's more readable/easier to understand?

Reply:
The output of `appstream-releases.py` is a fairly messy, multiline XML file which I'm not a fan of passing around as environment variables. And this script is really just a temporary workaround until something like [this PR](https://github.com/mozilla/mozillavpn-product-details/pull/93) lands - after which we can replace all of this with a static XML file that links to a hosted releases file.

The gist of the problem that led to this hack, is that we are required to provide a `<releases>` XML tag to pass Flatpak's submission guidelines - but we don't really have a good source of that release information. So we are trying to generate it dynamically from the git tags.

Reply:
The PR to mozillavpn-product-details has landed, so I have updated the PR to turn this into a static XML file that just links to the hosted [releases](https://mozilla.github.io/mozillavpn-product-details/org.mozilla.vpn.releases.xml) file.